cps/kni.c

```diff
@@ -546,14 +595,13 @@ route_cb(const struct nlmsghdr *nlh, void *arg)
 			rm->rtm_tos, rm->rtm_table, rm->rtm_protocol,
 			rm->rtm_protocol, rm->rtm_scope, rm->rtm_flags);
 
-	/*
-	 * TODO Which fields and attributes are needed for the LPM update?
-	 * Add them to @update here and in attr_get(), and if all of those
-	 * fields are present, set update->valid to true before returning.
-	 */
 	update->type = nlh->nlmsg_type;
 	update->family = rm->rtm_family;
 
+	/* Destination prefix length, e.g., 24 or 32 for IPv4. */
+	update->prefix_len = rm->rtm_dst_len;
+	update->oif_index = 0;
```

Comment:
1. It's better to add a blank line between these two lines because the comment only applies to the first line.

2. Add the following comment to the second line: `/* Default to an invalid index number. */`